QKSMS/src/main/java/com/moez/QKSMS/ui/view/ComposeView.java

```diff
@@ -753,7 +755,40 @@ public void saveDraft() {
 
                     // Only show the draft if we saved text, not if we just cleared some
                     if (!TextUtils.isEmpty(draft)) {
-                        Toast.makeText(mContext, R.string.toast_draft, Toast.LENGTH_SHORT).show();
+                        if (mRecipientProvider != null) {
+                            String[] addresses = mRecipientProvider.getRecipientAddresses();
+
+                            if (addresses != null && addresses.length > 0) {
+                                // save the message for each of the addresses
+                                for (int i = 0; i < addresses.length; i++) {
+                                    Calendar cal = Calendar.getInstance();
+                                    ContentValues values = new ContentValues();
+                                    values.put("address", addresses[i]);
+                                    values.put("date", cal.getTimeInMillis() + "");
```

Comment:
We can use `System.currentTimeMillis()` here, then we don't need a `Calendar` object